eslzArm/managementGroupTemplates/policyDefinitions/policies.json

```diff
@@ -7048,7 +7048,7 @@
                 },
                 {
                     "properties": {
-                        "description": "This policy denies the creation of a subsnet with out an Network Security Group. NSG help to protect traffic across subnet-level.",
+                        "description": "This policy denies the creation of a subnet with out an Network Security Group. NSG help to protect traffic across subnet-level.",
```

Comment:
"This policy denies the creation of a subnet without a Network Security Group (NSG). NSG helps to protect traffic across subnet-level."